intro-HPC/intro-HPC.tex

```diff
@@ -78,20 +78,20 @@ \part{Advanced Guide}
 
 
 % csub is only available to users in UGent & VUB clusters
-\ifgent
-\include{ch_checkpointing}
-\fi
-\ifbrussel
+\ifthenelse{\boolean{gent} \OR \boolean{brussel}}
+{
 \include{ch_checkpointing}
-\fi
+}{}
 
 \include{ch_program_examples}
 
 \include{ch_jobscript_examples}
 
 \include{ch_best_practices}
 
-\include{ch_VNC}
+\ifthenelse{\NOT \boolean{antwerpen}}{%
```

Comment:
@sbecuwe Can you add a comment here to explain why this chapter is excluded for Antwerp?

Reply:
OK, but please mention that in a comment above this line, to avoid that others re-introduce the VNC chapter for UA.

Reply:
Done. Also did it for checkpointing (gent and brussels).